### PR TITLE
docs: Avoiding conflict with dbt-athena-adapter

### PR DESCRIPTION
**Pull request by mrshu**
### Description

* Add a quick note on avoiding the conflict with `dbt-athena-adapter`.

* Fixes https://github.com/dbt-athena/dbt-athena/issues/103

Signed-off-by: mrshu <mrshu@cisco.com>

## Checklist
- [x] You followed [contributing section](https://github.com/dbt-athena/dbt-athena#contributing)
- [ ] You added unit testing when necessary
- [ ] You added functional testing when necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@
 * `pip install dbt-athena-community`
 * Or `pip install git+https://github.com/dbt-athena/dbt-athena.git`
 
+_**Note:** in order to avoid potential conflicts, make sure `dbt-athena-adapter` is not installed in the target environment._
```
Comment (mrshu) [inline README.md:35]: @nicor88 I am not really sure what this should so I'd be happy to take any suggestions here 🙂

Comment (nicor88) [reply README.md:35]: How about adding a note in https://github.com/dbt-athena/dbt-athena#known-issues ??

Comment (mrshu) [reply README.md]: @nicor88 sure, let's try that!

File: README.md [hunk 2]
```diff
@@ -225,6 +225,9 @@ The only way, from a dbt perspective, is to do a full-refresh of the incremental
 
 * Tables, schemas and database should only be lowercase
 
+* In order to avoid potential conflicts, make sure [`dbt-athena-adapter`](https://github.com/Tomme/dbt-athena) is not installed in the target environment.
```
Comment (mrshu) [inline README.md:228]: @nicor88 Once again, I am not totally sold on the wording so any suggestions are greatly appreciated 🙂

**Pull request merged**